css-transforms-2/Overview.bs

```diff
@@ -62,7 +62,7 @@ Module Interactions {#module-interactions}
 
 The <a>3D transform functions</a> here extend the set of functions for the 'transform' property.
 
-Some values of 'perspective', 'transform-style' and 'backface-visibility' result in the creation of a <a href="https://www.w3.org/TR/CSS2/visuren.html#containing-block">containing block</a>, and/or the creation of a <a spec="css2">stacking context</a>.
+Some values of 'perspective', 'transform-style' and 'backface-visibility' result in the creation of a [=containing block for all descendants=], and/or the creation of a <a spec="css21">stacking context</a>.
```

Comment:
This document doesnt' appear to define the term anywhere. (Unless it was already defined in some section that doesn't show up in this diff? Bikeshed doesn't seem to know the term, tho.) You can't rely on preceding versions of the document for a definition, it needs to be duplicated into here as well.

Reply:
The problem is that the definition lives in a previous version of the same shortname. Bikeshed has automatic logic to ignore previous versions of a spec, because that's nearly always what you want to do.  In particular, if a later version omits a definition that the previous version had, this was probably for a good reason! You thus don't want autolinking to continue to point to the old version; you want it to fail so you can notice the breakage and update things.

You want to make sure that the latest version of a given spec has all the definitions that you expect to need; in this case that means adding a `<dfn>` somewhere in *this* document.